wordpress-mobile/WordPress-Android
# Update sentry token and increase default Gradle memory to 6g

This PR brings back the `sentry.properties` file and updates the token used in it through mobile-secrets repo. It also increases the default Gradle memory to 6g as it's causing too many issues during release. Developers are still free to update it to whatever value they are comfortable with.

I've no idea why the auth token update was necessary. I can't find an integration for the token we were using, so for the new one I created a new internal integration in Sentry's dashboard and used that token so we can see and modify this token in the future.

**To test:**
* Run `bundle exec fastlane run configure_apply`
* Run `./gradlew buildVanillaRelease`
* Verify that app builds successfully

**PR submission checklist:**
